test/mpi/util/mtest.c

```diff
@@ -1385,6 +1385,31 @@ int MTestSpawnPossible(int *can_spawn)
     return errs;
 }
 
+/* ------------------------------------------------------------------------ */
+/* Utilities related to test environment */
+
+/* Some tests would like to test with large buffer, but an arbitary size may fail
+   on machines that does not have sufficient memory or the OS may not support it.
+   This routine provides a portable interface to get that max size.
+   It is taking the simpliest solution here: default to 2GB, unless user set via
+   environment variable -- MPITEST_MAXBUFFER
+*/
+MPI_Aint MTestDefaultMaxBufferSize()
+{
+    MPI_Aint max_size = 2147483648;
+    char *envval = NULL;
+    envval = getenv("MPITEST_MAXBUFFER");
+    if (envval) {
+        max_size = atol(envval);
+        if (max_size < 100000) {
+            fprintf(stderr, "MPITEST_MAXBUFFER (%s) is too small, we need at least 100MB\n",
+                    envval);
+            exit(1);
+        }
```

Comment:
The minimum size requirement seems somewhat arbitrary here.  I can understand if the minimum size be equal to the `basic type size * count`, but that information is not available here.  100MB seems somewhat arbitrary.  Can we remove that?

Reply:
What I meant to guard against is 0, which covers all cases of invalid numbers. Having a larger thresh provide more robustness of course. But I agree that 100MB is very arbitrary. What do you propose?

Reply:
I propose deleting that test.  If you like, you can check to make sure it's positive, but I don't care for that error check as much.

Reply:
OK.